core/src/main/java/hudson/scm/ChangeLogSet.java

```diff
@@ -205,6 +205,17 @@ public long getTimestamp() {
         @Exported
         public abstract User getAuthor();
 
+        /**
+         * Get from commit additional information
+         *
+         * <p>
+         * The exact definition depends on the individual SCM implementation
+         * @return
+         *     Can be empty
+         */
+        @Exported
+        public abstract String getComment();
```

Comment:
This is a breaking change. Any plugin with a class extending ChangeLogSet would be forced to implement this method when it's installed in a modern instance (https://github.com/search?p=2&q=org%3Ajenkinsci+%22+extends+ChangeLogSet%22&type=Code). The method should return a default value, maybe the `getMsg` method https://github.com/jenkinsci/jenkins/blob/be839e6606f77f1a5b3c8b60c9950c86b9f49879/core/src/main/java/hudson/scm/ChangeLogSet.java#L197